### PR TITLE
Add method of confirmation to linelist

### PR DESCRIPTION
**Pull request by attwad**

### CODE REVIEW

**Comment by attwad** [comment]
[Finalize columns of case list table](https://trello.com/c/gCPQL7a6/217-finalize-columns-of-case-list-table)

File: verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts [hunk 1]
```diff
@@ -9,6 +9,15 @@ describe('Linelist table', function () {
         cy.clearSeededLocations();
     });
 
+    it('Display case properly', function () {
+        cy.addCase('France', 'some notes', 'www.example.com');
+        cy.visit('/cases');
+        cy.contains('PCR test');
```
Comment (allysonjp715) [inline verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts:15]: It's not the best to test for something that wasn't explicitly set in the setup. WDYT about adding this to the addCase parameters? We could change the params to opts so people only have to set what they are explicitly testing for.

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 2]
```diff
@@ -389,9 +397,13 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                                 geoResolution:
                                                     c.location?.geoResolution,
                                                 locationName: c.location?.name,
-                                                confirmedDate: confirmedDate
-                                                    ? new Date(confirmedDate)
+                                                confirmedDate: confirmedEvent
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:400]: confirmedEvent?.dateRange?.start ? new Date...

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 3]
```diff
@@ -389,9 +397,13 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
...
+                                                    ? new Date(
+                                                          confirmedEvent.dateRange.start,
+                                                      )
                                                     : null,
+                                                confirmationMethod:
+                                                    confirmedEvent?.value || '',
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:406]: Is the `|| ''` needed?

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 2 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:400]: done

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 3 above)
Comment (attwad) [reply verification/curator-service/ui/src/components/LinelistTable.tsx:406]: ah no sorry that was before I used confirmedDate?: string as a type

File: verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts (hunk 1 above)
Comment (attwad) [reply verification/curator-service/ui/cypress/integration/components/LinelistTableTest.spec.ts:20]: good call, done.

**Pull request merged**